wdp/database/connector_wdp.py

```diff
@@ -0,0 +1,36 @@
+import sqlite3
+
+
+class Database:
+    def __init__(self, database_name="wdp_database.db"):
```

Comment:
In the future, you'd rather pass a `connection_string` to such an interface.
SQLite databases work in files located on the disk as well as in-memory: `:memory:`.
The most popular approach to that is setting an environment variable such as `DB_CONNECTION_STRING`, which is would be by default `wdp_database.db`, and then you read it using `os.getenv()`.

I mean, this is okay, just dropping a random curio.